js/script.js

```diff
@@ -0,0 +1,111 @@
+var Click = (function() {
+  this.numClicks = 0;
+  this.arr = [];
+});
```

Comment:
Also, I believe we discussed in class, but I would think about classes as being nouns rather than verbs. If this object represented information about a click itself (what was clicked, where on the page it was, etc.) the class name would make sense, but this is keeping track of other things. Maybe `GameKeeper` or something like that?